File: MPI/mpicheck/tests/functions/2.0/MPI_Type_set_name.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Datatype var_0;
    const char *var_1;
    int ret;
    /* calls */
    ret = MPI_Type_set_name(var_0, var_1);
    ret = PMPI_Type_set_name(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Type_set_name.f90

        
        program main
        use mpi
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
        call mpi_type_set_name(var_0, var_1, var_2)
       call pmpi_type_set_name(var_0, var_1, var_2)
        end program main
    